rpcs/ledgerService.go

```diff
@@ -259,6 +308,28 @@ func (ls *LedgerService) sendCatchupRes(ctx context.Context, target network.Unic
 	}
 }
 
+func topicBlockBytes(ledger *data.Ledger, round basics.Round, requestType string) network.Topics {
+	blk, cert, err := ledger.EncodedBlockCert(round)
+	if err != nil {
+		errMsg := "LedgerService topicBlockBytes: error in EncodedBlockCert: " + err.Error()
+		return network.Topics{
+			network.MakeTopic(network.ErrorKey, []byte(errMsg))}
```

Comment:
I think that you would want better error message, specifically for the `ledger.ErrNoEntry` case.
The rationale here is that it's very common to ask for a block where the other party doesn't have it.
In this case, returning an error such as `requested block is not available` would be a better error message.
( there is no "error" here )